helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -6,9 +6,11 @@ The values can be overridden in .Values.otelAgent.config
 extensions:
   health_check:
 
+{{- if .Values.autodetect.prometheus }}
   k8s_observer:
```

Comment:
I think we should still configure the observer by default as people have configured their own discovery rules to start things other than prometheus so I don't think it should be tied to this flag.

Reply:
sounds good. changed